gems/aws-sdk-s3/spec/plugins/md5s_spec.rb

```diff
@@ -17,7 +17,7 @@ module Plugins
           expect(client.config.compute_checksums).to be(false)
         end
 
-        it 'retries invalid content MD5s' do
+        it 'retries invalid content MD5s', rbs_test: :skip do
```

Comment:
When I remove this, the test passes, and rbs:spytest passes. Are these no longer necessary, or am I missing some information?

Reply:
It still fails in my environment.
This is because this test is an error pattern.
Are files generated in gems/aws-sdk-s3/sig?

```
$ RUBYOPT="-r bundler/setup -r rbs/test/setup" RBS_TEST_RAISE=true RBS_TEST_LOGLEVEL=error RBS_TEST_OPT="-I gems/aws-sdk-core/sig -I gems/aws-sdk-s3/sig" RBS_TEST_TARGET="Aws,Aws::*,Seahorse,Seahorse::*" bundle exec rspec gems/aws-sdk-s3/spec --tag '~rbs_test:skip'
Run options: exclude {:rbs_test=>"skip"}
........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................F............................................................................................

Failures:

  1) Aws::S3::Plugins::Md5s retries invalid content MD5s
     Failure/Error: resp = s3.put_object(bucket: 'test', key: 'test', content_md5: '456==')

     RBS::Test::Tester::TypeError:
       TypeError: [Aws::S3::Client#put_object] UnresolvedOverloadingError: couldn't find a suitable overloading
     # ./gems/aws-sdk-s3/spec/plugins/md5s_spec.rb:46:in `block (2 levels) in <module:Plugins>'

Finished in 3.3 seconds (files took 1.03 seconds to load)
1141 examples, 1 failure

Failed examples:

rspec ./gems/aws-sdk-s3/spec/plugins/md5s_spec.rb:20 # Aws::S3::Plugins::Md5s retries invalid content MD5s

$ bundle exec rbs --version
rbs 3.4.1
```